src/index.ts

```diff
@@ -141,116 +149,62 @@ Examples of user prompts in which you should use this plugin:
 "Show me a diagram of ... ."
 "Show me how UIView lifecycle works."
 
-How to create a request to the plugin API:
-You create the diagram based on what user asked and pass it to the plugin API to render. Mermaid is the preferred language.
-
-Important rules when creating the diagram in mermaid syntax:
-- Prefer using graph TB types of diagrams.
-- Avoid linear diagrams when possible, diagrams should be hierarchical and have multiple branches when applicable.
-- Never use the ampersand (&) symbol in the diagram, it will break the diagram. Use the word "and" instead. For example use "User and Admin" instead of "User & Admin".
-- Never use round brackets () in the node identifiers, node labels and edge labels, it will break the diagram. Use a coma instead. For example use "User, Admin" instead of "User (Admin)".
-- Don't use empty labels "" for edges, instead don't label the edge at all. For example U["User"] --> A["Admin"].
-- Don't add the label if its the same as the destination node.
-
-Rules when using graph diagrams in mermaid syntax:
-- Use short node identifiers, for example U for User or FS for File System.
-- Always use double quotes for node labels, for example U["User"].
-- Always use double quotes for edge labels, for example U["User"] -- "User enters email" --> V["Verification"].
-- Indentation is very important, always indent according to the examples below.
-
-Rules when using graph diagrams with subgraphs in mermaid syntax:
-Never refer to the subgraph root node from within the subgraph itself.
-
-For example this is wrong subgraph usage:
-\`\`\`
-graph TB
-  subgraph M["Microsoft"]
-    A["Azure"]
-    M -- "Invested in" --> O
-  end
-  
-  subgraph O["AI"]
-    C["Chat"]
-  end
-\`\`\`
-
-In this diagram M is referenced from within the M subgraph, this will break the diagram.
-Never reference the subgraph node identifier from within the subgraph itself.
-Instead move any edges that connect the subgraph with other nodes or subgraphs outside of the subgraph like so.
-
-Correct subgraph usage:
-\`\`\`
-graph TB
-  subgraph M["Microsoft"]
-    A["Azure"]
-  end
-
-  M -- "Invested in" --> O
-  
-  subgraph O["OpenAI"]
-    C["ChatGPT"]
-  end
-\`\`\`
-
-Examples of invoking the plugin API:
```

Comment:
I understand you have moved the examples to the guidance, which makes sense for other diagrams, but for graph TB maybe we should include the guidance inline. Maybe we can pull it from the guidance map, since that provides better organization